tests/unit/src/commands/clash.spec.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 var mtgClash = require('../../../../src/commands/clash'),
-    mtg = requre('mtgsdk'),
+    mtg = require('mtgsdk'),
```

Comment:
good catch. I'm confused as to why this didn't break when I ran the tests before ...